### ISSUE TITLE
Refresh CORD-19 annotations with latest scibite annotations

### ISSUE THREAD

**Issue by cmungall** [Knowledge-Graph-Hub/kg-covid-19#261]
https://github.com/SciBiteLabs/CORD19/issues/14

Note they use PRO as the vocabulary. We can ingest the PRO file... but it may be easier just to add the PRO xrefs into our curated gpi file

**Comment by justaddcoffee** [comment]
@cmungall @deepakunni3 does SciBite definitely use PRO as their vocabulary for SARS-CoV-2 genes? I can't seem to find for example the PRO ID for Spike protein:
https://proconsortium.org/app/entry/PR%3A000050269/
in our transformed data:

```
$ grep 000050269 data/transformed/SciBite-CORD-19/nodes.tsv  data/transformed/SciBite-CORD-19/edges.tsv
$
```

**Comment by justaddcoffee** [comment]
Closed by #263
